ha/k8s_setup/ha_setup.py

```diff
@@ -178,7 +178,7 @@ def process(self):
             # Ref ticket EOS-25694
             data_pod_label = Conf.get(self._index, f'cortx{_DELIM}common{_DELIM}product_release')
             # TBD delete once data_pod_label is avilable from confstore
-            data_pod_label = 'cortx-data'
+            data_pod_label = ['cortx-data', 'cortx-server']
```

Comment:
Currently these are hard coded but I assume that this information will come from Conf Store eventually?